README.md

```diff
@@ -194,3 +195,20 @@ Prepare your environment for the installation commands.
 ## Proxy support
 
 [Configuring proxy support for ExternalDNS Operator](./docs/proxy.md)
+
+## Metrics
+
+The ExternalDNS Operator exposes [controller-runtime metrics](https://book-v1.book.kubebuilder.io/beyond_basics/controller_metrics.html) in the format expected by [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator).
```

Comment:
Maybe link to the latest version of the [kubebuilder book](https://book.kubebuilder.io/reference/metrics.html)